### PR TITLE
Add filter by language to CU lists (AR-677)

### PR DESCRIPTION
**Pull request by davgur**

### CODE REVIEW

File: api/handlers.go [hunk 1]
```diff
@@ -2270,8 +2268,23 @@ func appendMediaLanguageFilterMods(exec boil.Executor, mods *[]qm.QueryMod, f Me
...
+				"AND f.secure = 0 AND f.published IS TRUE AND f.language = ?))", f.MediaLanguage),
+		)
+	}
+	/*	where := fmt.Sprintf("(type_id = %[1]d AND properties->>'original_language' = '%[2]s') OR "+
+		"(id in ( SELECT DISTINCT cu.id FROM content_units cu "+
+		"INNER JOIN files f "+
+		"ON f.content_unit_id = cu.id AND cu.secure = 0 AND cu.published IS TRUE "+
+		"AND f.secure = 0 AND f.published IS TRUE AND f.language = '%[2]s'))",
+		mdb.CONTENT_TYPE_REGISTRY.ByName[consts.CT_BLOG_POST].ID, f.MediaLanguage)
+	*mods = append(*mods, qm.Where(where), )*/
```
Comment (davgur) [inline api/handlers.go]: I not sure witch solution is better:
1 - SQL query for necessary data
2 not need additional parameter (ContentTypes)

**Comment by davgur** [comment]
[Score of Tweets is nil when sorting is not by relevance](https://trello.com/c/Qdu8645K/8-score-of-tweets-is-nil-when-sorting-is-not-by-relevance)

**Comment by alexmbb** [comment]
@davgur and @gshilin 
is this pr ready?

File: api/handlers.go [hunk 2]
```diff
@@ -2270,8 +2268,12 @@ func appendMediaLanguageFilterMods(exec boil.Executor, mods *[]qm.QueryMod, f Me
 		return nil
 	}
 
-	*mods = append(*mods, qm.Where("properties->>'original_language' = ?", f.MediaLanguage))
-
+	*mods = append(*mods,
+		qm.WhereIn("(id in ( SELECT DISTINCT cu.id FROM content_units cu "+
```
Comment (edoshor) [inline api/handlers.go:2272]: please use ` for multiline strings in go instead of +

Comment (edoshor) [reply api/handlers.go:2272]: Please add a TODO comment saying this query should be optimized ASAP and before we do that clients should use it as little as possible.

Comment (davgur) [reply api/handlers.go]: [commit](https://github.com/Bnei-Baruch/archive-backend/pull/160/commits/671c9c5a75f9e22c75361769d43c7c71f89bf6df)

**Comment by alexmbb** [comment]
@gshilin Can we merge it to master?

**Comment by gshilin** [comment]
That's Edo

**Pull request merged**